spec/integration/client/client_spec.rb

```diff
@@ -56,7 +58,18 @@ def stop_tiny_server
 cookbook_path "#{path_to('cookbooks')}"
 EOM
 
-      result = shell_out("#{chef_client} -c \"#{path_to('config/client.rb')}\" -o 'x::default'", :cwd => chef_dir)
+      result = shell_out!("#{chef_client} -c \"#{path_to('config/client.rb')}\" -o 'x::default'", :cwd => chef_dir)
+    end
+
+    it "should complete successfully with no other environment variables", :skip => (Chef::Platform.windows?) do
+      file 'config/client.rb', <<EOM
+local_mode true
+cookbook_path "#{path_to('cookbooks')}"
+EOM
+
+      Chef::Log.info "Bare invocation will have the following load-path."
+      Chef::Log.info shell_out!("env -i #{critical_env_vars} ruby -e 'puts $:'").stdout
```

Comment:
Not necessarily - but if that test fails (on some CI environment or elsewhere), I would have no idea how to debug it without either jumping on the box and figuring out the environment or looking at that.  I figured I'd just help my future self by leaving the command to dump the lookup path in there.